### ISSUE TITLE
UI issues for new users

### ISSUE THREAD

**Issue by ghost** [publicdomaincompany/simoji#9]
The UI looks like a web based game.

The Play button at the bottom is far far below.

The play button also does not seem to autopause when i change from buisness to moths to forest fire.

Is it intentional. because there seems to be no autopause but changing from one to another auto resets the simulation to start state.

**Comment by breck7** [comment]
Yes, the no auto-paused is intentional (so you can flip through examples better).

Good feedback on the Play button. I can make that better.

I'm not actually sure if I'll continue with this. My concept is sometimes when I'm writing an essay I want to explore an idea using a multi-agent simulation, and I'm not good enough with any of the existing tools so I've been exploring building my own. It is a fun programming exercise as learning about game loops and quadtrees and collision detection is not the kind of thing I normally do. But I'm not sure long term whether to continue with this. Every now and then I get the urge to get it to the next level.
